src/components/QueryCondition.vue

```diff
@@ -19,12 +19,19 @@
 			:placeholder="$i18n('query-builder-input-value-placeholder')"
 			:disabled="isTextInputDisabled()"
 		/>
+		<Button
```

Comment:
This should probably be its own simple subcomponent as well, similar to ValueTypeDropdown and PropertyLookup. The idea is to try to maintain "one level of abstraction" within the template as well.

But that could also be done in a follow-up pull request

Reply:
Let's do it in a followup. :)

Reply:
:+1: